api/proto/proto.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package proto;
+option go_package = "./proto";
+
+message Request {
+  bytes payload = 1;
+}
+
+message Response {
+  string error = 1;
```

Comment:
Does this even return? Is there a better way to check for errors?

Reply:
Errors in grpc are first class citizens. Fixed like: https://github.com/protocolbuffers/protobuf/blob/0c21b63cb7903460696080d0d4fbdfd42525db5e/src/google/protobuf/empty.proto#L48 in https://github.com/narvikd/nubedb/pull/10/commits/528f96871c0ee30456db7899a119f338dff4e18c